README.md

```
@@ -73,6 +82,78 @@ Now ask your AI to debug PHP code.
+## Try It Out
+
+### 1. Check Xdebug Installation
+
+**Recommended Setup:** Xdebug installed but disabled in php.ini (loaded on-demand for zero performance impact)
+
+```bash
+./bin/check-env
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)